Dockerfile

```diff
@@ -25,8 +25,6 @@ RUN USER=root cargo new --bin ccconsensus
 WORKDIR /usr/src/ccconsensus
 
 COPY Cargo.toml Cargo.lock ./
-RUN cargo build --release && \
-    rm src/*.rs
```

Comment:
If you aren't compiling the binaries here then where would they come from ?  Jira ticket doesn't have any description why that is.

Reply:
Correct me if I'm wrong, there are two cargo build commands. The first one is unnecessary.

Reply:
I see it now. Thanks.